EasyPost.Net/Client.cs

```diff
@@ -40,31 +41,40 @@ public int RequestTimeoutMilliseconds
         ///     Constructor for the EasyPost client.
         /// </summary>
         /// <param name="clientConfiguration">EasyPost.ClientConfiguration object instance to use to configure this client.</param>
-        public Client(ClientConfiguration clientConfiguration)
+        /// <param name="overrideHttpClient">Custom HttpClient to pass into RestSharp if needed. Mostly for debug purposes, not advised for general use.</param>
```

Comment:
Can we call this `customHttpClient` instead? We've had other libs start doing this where users can pass in their own client. May not make as much sense in C# but at least we can stay consistent.

Reply:
Resolved in d6e130a215e674be1d75eca6463090284a256f15